temel_bilgiler/kurulumlar/bilgisayar_temelleri.md

```diff
@@ -0,0 +1,44 @@
+### Giriş
+
+Söylemeye gerek yok, kodlamayı öğrenirken zamanınızın çoğunu bilgisayar başında geçireceksiniz. Bu nedenle, bilgisayarınızla nasıl çalışacağınızı anlamak çok önemlidir. Bu derste, sizi hızlı bir şekilde bilgisayar temelleri konusunda bilgilendirecek bir kaynak tanıtacağız.
+
+### Derse Genel Bakış
+
+Bu bölüm, bu derste öğreneceğiniz konulara genel bir bakış içerir.
+
+* Bilgisayar yazılımı ve donanımı
+* Ekran görüntüleri nasıl alınır
+* Güçlü parolaların önemi
+
+### Ödev
+
+<div class="lesson-content__panel" markdown="1">
+
+İyi niyet topluluğu vakfı (GCF) kaynakları, temel bilgileri öğrenmeniz için mükemmeldir. Vakfın eşlik eden videoları, çizimleri vardır ve okunması kolaydır. Aşağıdaki okumalar yeterli olacaktır, ancak GCF'nin diğer kaynaklarına da bakabilirsiniz.
+
+1. [What is a computer?](https://edu.gcfglobal.org/en/computerbasics/what-is-a-computer/1/)
+2. [What is an operating system?](https://edu.gcfglobal.org/en/computerbasics/understanding-operating-systems/1/)
+3. [What is an application?](https://edu.gcfglobal.org/en/computerbasics/understanding-applications/1/)
+4. [Open source vs closed source software](https://edu.gcfglobal.org/en/basic-computer-skills/open-source-vs-closed-source-software/1/)
+5. [Taking screenshots](https://edu.gcfglobal.org/en/techsavvy/taking-screenshots/1/)
+6. [Creating strong passwords](https://edu.gcfglobal.org/en/techsavvy/password-tips/1/)
+
+</div>
+
+
+### Bilgi kontrolü
+
+Bu bölüm, bu dersi anlayıp anlamadığınızı kendi başınıza kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta sorun yaşıyorsanız, soruya tıklayın ve bağlantının verdiği materyali inceleyin.
+
+* [Is Windows an operating system or an application?](https://edu.gcfglobal.org/en/computerbasics/understanding-operating-systems/1/)
+* [What's the difference between open source and closed source software?](https://edu.gcfglobal.org/en/basic-computer-skills/open-source-vs-closed-source-software/1/)
+* [What are some use cases of a screenshot?](https://edu.gcfglobal.org/en/techsavvy/taking-screenshots/1/)
+* [Give an example of a weak and a strong password.](https://edu.gcfglobal.org/en/techsavvy/password-tips/1/)
+
+### Ek kaynaklar
+
+Bu bölüm, ilgili içeriğe yardımcı olacak bağlantılar içerir. Zorunlu değil, ek kaynak olarak düşünün.
```

Comment:
Let's make this line standard too. Yours are not wrong translations obviously, but it's nice to have same translation for all. Sorry for these, I will inform other volunteers as well.

fix: Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.